.github/workflows/kickstart-tests.yml

```diff
@@ -79,6 +79,21 @@ jobs:
           echo "workflow arguments are: $ARGS"
           echo "comment_args=${ARGS}" >> $GITHUB_OUTPUT
 
+      - name: Get timestamp
+        id: timestamp
+        run: echo "::set-output name=timestamp::$(date +'%Y-%m-%d_%H:%M:%S')"
+
+      - name: Construct image description
+        id: kstest_run_description
+        run: |
+          set -eux
+          # comment on PR
+          pr_num="${{ github.event.issue.number }}"
+          timestamp=${{ steps.teimstamp.outputs.timestamp }}
+          sha="${{ steps.pr_api.outcome == 'success' && fromJson(steps.pr_api.outputs.data).head.sha }}"
+          echo "image_description=pr$pr_num-$timestamp-$sha" >> $GITHUB_OUTPUT
```

Comment:
I realize this can be nicely made reusable role among this workflow and the build-image workflow.